web/src/emoji_picker.js

```diff
@@ -189,21 +187,22 @@ function refill_section_head_offsets($popover) {
 }
 
 export function reactions_popped() {
-    return $current_message_emoji_popover_elem !== undefined;
+    return emoji_popover_instance;
 }
 
 export function hide_emoji_popover() {
-    $(".has_popover").removeClass("has_popover has_emoji_popover");
+    current_message_id = null;
     if (user_status_ui.user_status_picker_open()) {
         // Re-enable clicking events for other elements after closing
         // the popover.  This is the inverse of the hack of in the
         // handler that opens the "user status modal" emoji picker.
         $(".app, .header, .modal__overlay, #set-user-status-modal").css("pointer-events", "all");
     }
     if (reactions_popped()) {
-        $current_message_emoji_popover_elem.popover("destroy");
-        $current_message_emoji_popover_elem.removeClass("reaction_button_visible");
-        $current_message_emoji_popover_elem = undefined;
+        $(emoji_popover_instance.reference).removeClass("reaction_button_visible");
+        $(emoji_popover_instance.reference).parent().removeClass("reaction_button_visible");
+        emoji_popover_instance.destroy();
```

Comment:
Also this doesn't seem to be getting triggered as I am seeing multiple instances of the emoji popover:
<img width="1207" alt="image" src="https://github.com/zulip/zulip/assets/25124304/38bcf623-babe-4aae-8578-c7966f222984">